### DISCUSSION TITLE
Addresses: Set as default option / name is displayed twice on the cards

### DISCUSSION THREAD

**Discussion by davidhellmann** [craftcms/cms#14042] [Ideas]
It's just a little one but it would be great to have an option, like Shopify and other systems to set an address as default. Is there just one address it's the default one by default.

Second is when you add the `Full Name` Field to the addresses it shows the Name twice on cards. Would be great to show just the User Name and if the Full Name has a value, show this one. 

![image](https://github.com/craftcms/cms/assets/764292/757f1069-b154-4841-b351-7afbf6399aa9)

**Comment by brandonkelly** [comment]
Craft CMS itself doesn’t actually use addresses for anything, so this would be a little confusing (“Default for what?”).

Craft Commerce adds the ability to set a default billing address and a default shipping address, which would also add to the confusion if there’s also a generic “default” address.

Regarding the name getting output twice, you probably have “Include this field in element cards” selected within your “Full Name” field’s settings in the address field layout. The name is included in formatted addresses, so that’s redundant if you also have “Include this field in element cards” selected for the “Address” field.
